### ISSUE TITLE
Can't configure ports property to include hostPort in Kind: Deployment

### ISSUE THREAD

**Issue by ZakariaNaaija** [kubernetes/kompose#1545]
Specifying 
```
    ports:
      - target: 80
        published: 80
        protocol: tcp
        mode: host
```

In docker-compose should be converted it using Kompose to a Deployment resource as
```
    ports:
      - containerPort: 80
        hostPort: 80
        protocol: TCP
```

But unfortunately the result is:
```
    ports:
      - containerPort: 80
```
Or is there any way to expose a service outside the cluster using 80 and 443 ports?

**Comment by ZakariaNaaija** [comment]
Same with https://github.com/kubernetes/kompose/issues/1054

**Comment by AhmedGrati** [comment]
Closed as resolved by #1558.

**Comment by cdrage** [comment]
@ZakariaNaaija @AhmedGrati Sorry for getting back to this after 2 years...

Unfortunately this is something we should of not supported and instead used a label instead for being able to provide using a hostPort. 

This is totally on me and I overlooked it.
